src/windows/video_widget.py

```diff
@@ -82,7 +82,7 @@ def paintEvent(self, event, *args):
         painter = QPainter(self)
         painter.setRenderHints(QPainter.Antialiasing | QPainter.SmoothPixmapTransform | QPainter.TextAntialiasing, True)
 
-        # Fill background black
+        # Fill background with theme color instead of black
         painter.fillRect(event.rect(), self.palette().window())
```

Comment:
With `Qt.WA_OpaquePaintEvent` set, it _may_ not even be necessary to do this manual `fillRect()`. I think the widget should be filled with its background color by default. (And if it is redundant, might as well avoid it, since we're going to be repainting over most of the area with the video preview itself.)

Reply:
Eep! Never mind, I just tried it and I am _very_ wrong about that. The painting is a mess without the `fillRect()`,

Still, maybe change the comment to just "Fill widget area with theme color"? ("instead of black" sounds so weirdly specific. It's "instead of white" or "instead of purple", too, but listing all the colors it's _not_ would be tedious.)